pkg/operator/ceph/cluster/watcher.go

```diff
@@ -78,7 +78,7 @@ func (c *clientCluster) onK8sNode(object runtime.Object) bool {
 		return false
 	}
 
-	valid, _ := k8sutil.ValidNode(*node, cluster.Spec.Placement.All())
+	valid, _ := k8sutil.ValidNode(*node, cephv1.GetOSDPlacement(cluster.Spec.Placement))
```

Comment:
If there is no osd placement specified in the CR, this will still result in all nodes being valid, right?

Reply:
Yes, as is done within the `GetOSDPlacement()` function: https://github.com/rook/rook/blob/3ac06a533682eb5e40f954cf7255a9d28fb98ed7/pkg/apis/ceph.rook.io/v1/placement.go#L32-L35